sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -491,24 +471,25 @@ Bound withTestServices(BigQueryServices testServices) {
 
       @Override
       public void validate(PInput input) {
-        if (validate) {
```

Comment:
I ended up needing it here. Here is why:

1. We were previously checking that we couldn't apply a BigQueryIO operation without specifying at least a query or table.
2. We were doing so using .named, which we can no longer do.
3. To preserve those tests, I tried ".withoutValidation"
4. This leads to a NPE when constructing the BigQuery read operation, because we *don't* validate that at least one of the table or query is present. Instead, we construct the operation, and then the constructor has a Precondition.checkNotNull that fails.

My understanding is that withoutValidation should prevent the "check with BigQuery to make sure the table/query is correct" (eg., because an earlier stage of the pipeline may produce the table, so we expect it to fail). That doesn't mean we should skip validating the transform is correctly constructed.

TL;DR "NullPointerException: table" is less informative than "You must specify a query or table"